crates/ark/src/interface.rs

```diff
@@ -774,6 +780,13 @@ impl RMain {
         if info.browser {
             match self.dap.stack_info() {
                 Ok(stack) => {
+                    if let Some(frame) = stack.first() {
```

Comment:
I just want to point out that if you click on a different call stack frame in the UI, then you obviously only still get completions for the _first_ frame in the stack, because that's the position R is actually sitting at, even though you've walked up the stack in the UI.

I think that is totally expected and fine, it's just a possible source of confusion